evm/src/cpu/gas.rs

```diff
@@ -99,6 +98,15 @@ fn eval_packed_accumulate<P: PackedField>(
     let ternary_op_cost = P::Scalar::from_canonical_u32(G_MID.unwrap())
         - lv.opcode_bits[1] * P::Scalar::from_canonical_u32(G_MID.unwrap());
     yield_constr.constraint_transition(lv.op.ternary_op * (nv.gas - lv.gas - ternary_op_cost));
+
+    // For JUMPDEST and KECCAK_GENERAL.
+    // JUMPDEST is differentiated from KECCAK_GENERAL by its second bit set to 1.
+    let jumpdest_keccak_general_gas_cost = lv.opcode_bits[1]
+        * P::Scalar::from_canonical_u32(G_JUMPDEST.unwrap())
```

Comment:
These `unwrap`s are not great. I know they are not new in this PR but it seems to me like using `Option` in `SIMPLE_OPCODES` has had some unfortunate consequences. Perhaps it should be refactored to use an enum so we get `OpsColumnsView<ManualOrCode>`, with `enum ManualOrCode { Manual, Code(u32) }`?
Alternatively, have two collections, one for manually managed opcodes and the other for simple opcodes and go through each separately?

Reply:
They need to be wrapped in an `Option`, because we have a distinction here between `Some(0)` and `None`, though they are pretty harmless as we only `unwrap` hardcoded constants so it doesn't really bother me to rely on unwrap here.

Reply:
To me having to resort to using `unwrap` is a code smell. Would having two hard-coded lists (manual vs "simple") work?